kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -197,7 +197,7 @@ def exec_prove_legacy(
     bug_report: bool = False,
     save_directory: Path | None = None,
     spec_module: str | None = None,
-    claim_labels: Iterable[str] = (),
+    claim_labels: Iterable[str] | None = None,
```

Comment:
Why this change?

Reply:
`claim_labels` is passed directly to `KProve.get_claims`, where `()` is interpreted as "filter out all claims". So if you call without explicitly specifying all labels, something will be left out.

When calling through the CLI this bug does not occur, as the `Namespace` contains `claim_labels=None`, which in `get_claims` gets interpreted as "filter out no claims". This is obviously violating the original type hint, but the part of code where this value comes from is practically not type checked.